src/app-layer-protos.h

```diff
@@ -101,6 +109,48 @@ static inline bool AppProtoEquals(AppProto sigproto, AppProto alproto)
     return false;
 }
 
+// whether a signature AppProto matches a flow (or signature) AppProto
```

Comment:
can you add a comment here on the interplay between doh2 and http2 and sigs using either?

Reply:
ok